.storybook/config.js

```diff
@@ -1,5 +1,16 @@
 import {configure} from '@storybook/html';
+import { twig } from 'twig';
 
 const context = require.context('../src', true, /\.stories\.js$/);
+const twigContext = require.context('../src/patterns', true, /\.twig$/);
+
+twigContext.keys().forEach(key =>
```

Comment:
This looks to be causing an error by being run multiple times on the page.

Reply:
Took a while to replicate, but it only happens when changing a pattern and it's rebuilt.

Reply:
Going to crack on and merge this PR so we can use Twig in the trial. Will raise a ticket for investigating this issue further.

Reply:
For reference, an example of the error in the browser console looks like this:
<img width="1224" alt="Screenshot 2019-10-03 17 50 31" src="https://user-images.githubusercontent.com/2893480/66147250-66c6af00-e606-11e9-9c26-cdec092b12f8.png">